samples/cpp/benchmark_app/main.cpp

```diff
@@ -407,14 +410,14 @@ int main(int argc, char* argv[]) {
             slog::info << "Loading network files" << slog::endl;
 
             auto startTime = Time::now();
-            CNNNetwork cnnNetwork = ie.ReadNetwork(FLAGS_m);
+            auto model = core.read_model(FLAGS_m);
```

Comment:
The private VPUX jenkins failed with both : blobs and IRs.
Error messages:
* `Please check that model format: xml is supported and the model is correct.`
* `Please check that model format: blob is supported and the model is correct.`

Reply:
Those messages are not from BA, looks like OV throws an exception. Could you check where exactly it fails (as I don't have VPUX machine to debug) ? Please attach stack trace if it is available.

Reply:
It is not VPUX specific issue. I see you rebased your PR, let me re-start the jobs:

* https://dsp-ci-icv.inn.intel.com/job/Nets-Validation/job/manual/job/Yocto/6175/
* https://dsp-ci-icv.inn.intel.com/job/Nets-Validation/job/manual/job/Windows/1243/

Reply:
Alright. It didn't help. There are frontends available but still, models cannot be read. Compile_tool can read the models perfectly though... That's really weird. Let me check if it is reproducible locally.

Reply:
It works locally. Looks like, we need to reproduce the environment of nets-validation jobs.
@vinograd47 @mznamens 

if you have any ideas what is wrong, let us know